_data-prepper/pipelines/configuration/processors/trace-peer-forwarder.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: Trace peer forwarder
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# Trace peer forwarder
+
+The `Trace peer forwarder` processor is used to reduce the number of events that are forwarded in a [Trace analytics]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/trace-analytics/) pipeline by half when using [Peer forwarder]({{site.url}}{{site.baseurl}}/data-prepper/managing-data-prepper/peer-forwarder/). Pipelines peer forward events, which causes the core peer forwarder to make multiple HTTP requests for the same event. You can use `Trace peer forwarder` to forward the event one time in the entry pipeline (`otel-trace`) instead of other pipelines, such as `service-map-pipeline` and `raw-pipeline`, and prevents unnecessary HTTP requests. `Trace peer forwarder` groups events based on `trace_id` similar to the `service_map_stateful` and `otel_trace_raw ` processors. 
+
+You should use `Trace peer forwarder` when you have multiple nodes, and for trace analytics pipelines.
+
+In [Trace analytics]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/trace-analytics/), each event is duplicated when it is sent from the `otel-trace-pipeline` to the `raw-pipeline` and to the `service-map-pipeline`. The event is forwarded once in each pipeline. When you use this processor, the event is forwarded once in the `otel-trace-pipeline` pipeline to the correct peer. 
```

Comment:
You provided the link to Trace analytics in the first section above. Doesn't seem necessary here.
In fact, this section (line 15) seems redundant. I think you might roll it in with the description above.

"The Trace peer forwarder processor is used with [Peer forwarder](https://github.com/opensearch-project/documentation-website/pull/3238/%7B%7Bsite.url%7D%7D%7B%7Bsite.baseurl%7D%7D/data-prepper/managing-data-prepper/peer-forwarder/) to reduce by half the number of events forwarded in a [Trace analytics](https://github.com/opensearch-project/documentation-website/pull/3238/%7B%7Bsite.url%7D%7D%7B%7Bsite.baseurl%7D%7D/data-prepper/common-use-cases/trace-analytics/) pipeline."

"Typically in Trace analytics, each event is duplicated when it is sent from `otel-trace-pipeline` to `raw-pipeline` and `service-map-pipeline`.

"When pipelines forward events, this causes the core peer forwarder to make multiple HTTP requests for the same event."

"You can use Trace peer forwarder to forward an event once through the `otel-trace-pipeline` instead of `raw-pipeline` and `service-map-pipeline`, thereby preventing unnecessary HTTP requests."

Then you could just deleted line 15. But in this fragment, "... this causes the core peer forwarder to make multiple HTTP requests ...", I would check with an SME to make sure "multiple" is correct, or should it be "duplicate"?